CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ You can find its changes [documented below](#022-2025-01-03).
 
 This release has an [MSRV][] of 1.82.
 
+### Added
+
+* Support for the ACES2065-1 color space. ([#124][] by [@tomcur][])
```

Comment:
Need link below. Since I too love to forget them...

Reply:
Too easy to forget!

Reply:
My settings are:
```
"markdownlint.config": {
        "no-emphasis-as-heading": false,
        "ul-indent": false,
        "no-inline-html": false,
        "first-line-h1": false,
        "no-duplicate-heading": false
    },
```